.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.2
```

Comment:
Apologies, I haven't been keeping up with the image releases; does this mean we can upgrade to Ruby 2.6.2 and still run Circle?

Reply:
Circle released/updated 2.6.2 tagged images on [Docker Hub](https://hub.docker.com/r/circleci/ruby/tags) this morning.